app-next/components/MealsList/MealsList.jsx

```diff
@@ -0,0 +1,32 @@
+ "use client"
+import { useEffect, useState } from "react";
+
+export const MealsList = () => {
+    const [meals, setMeals] = useState([]);
+    const apiUrl = process.env.NEXT_PUBLIC_API_URL || "/api/meals";
```

Comment:
If apiUrl is already set to "/api/meals", this results in "/api/meals/api/meals"